### ISSUE TITLE
Nuscenes validate set

### ISSUE THREAD

**Issue by Liuxinyv** [pengsongyou/openscene#17]
Thanks for sharing your data and code, your work has been very helpful to me!
I noticed that you updated the point cloud data for the nuscenes validation set. The new point clouds contain approximately 300,000 points each, whereas the original lidar point cloud only contained about 30,000 points. Were these additional 300,000 points obtained by concatenating neighboring frames of the nuscenes data? If I want to retrain the model, do I also need to update the lidar point cloud in the training set?

**Comment by pengsongyou** [comment]
Yes, you are right. Quoting what we wrote in Appendix A:

> For nuScenes, we input all Lidar points within the half-second segments, and only train with point features at the last time stamp.

Before we released the wrong Lidar point clouds which are only the points on the last timestamp, so the input and output points are the same. Now, the new ones are all points of half-second segments as the input points (~300K points), which gives more information, and train only with the point features on the points of last timestamp (~30K points). We observed better performance in this way.

Best
Songyou
